### ISSUE TITLE
Create User/Group Management in the GUI

### ISSUE THREAD

**Issue by wirecutter313** [retspen/webvirtmgr#226]
It would be really cool to see a User and RBAC page to create and give access to users.

**Comment by geetanshjindal** [comment]
just want to know is there any possible way to combine "webvirtmgr" and "webvirtcloud" ?

**Comment by CamJGaming** [comment]
already been asked.......

**Comment by martinsefcik** [comment]
Guys you mentioned webvirtcloud a year ago, but webvirtclound was pushed to the repository just few days ago (see https://github.com/retspen/webvirtcloud) So I'm confused, and also confused what's the difference between these two apps. Will webvirtmgr deprecated or what ?
@retspen Can you please explain it ?

**Comment by sammcj** [comment]
Yeah I was wondering this too, https://github.com/retspen/webvirtcloud has no description / screenshots or anything too?

**Comment by marukochanstella** [comment]
@retspen You can exaplain it ?
